lib/utils.js

```diff
@@ -162,7 +162,7 @@ const hasPullRequestBeenApproved = async (context, pullRequestNumber) => {
     })
   );
 
-  return searchResult.status === SUCCESS_STATUS;
+  return searchResult.data.items.length > 0;
```

Comment:
Explain with a comment why we are checking like this (same as what we discussed offline.)

Reply:
Also, if the constant `SUCCESS_STATUS` is not being used, remove it.

Reply:
`SUCCESS_STATUS` is used for checking if the user is a collaborator.

> Explain with a comment why we are checking like this (same as what we discussed offline.)

Done.